src/components/Calculator.js

```diff
@@ -1,34 +1,73 @@
-        <div className="int_values ">1</div>
-        <div className="int_values ">2</div>
-        <div className="int_values ">3</div>
-        <div className="arithmetic_operand">&#215;</div>
-        <div className="int_values ">5</div>
-        <div className="int_values ">6</div>
-        <div className="int_values ">7</div>
-        <div className="arithmetic_operand">&#x2D;</div>
-        <div className="int_values ">1</div>
-        <div className="int_values ">2</div>
-        <div className="int_values ">3</div>
-        <div className="arithmetic_operand">&#43;</div>
-        <div className="int_values zero">0</div>
-        <div className="int_values ">.</div>
-        <div className="arithmetic_operand">=</div>
+        <div className="calc_display">{ this.displayResult() }</div>
+        <div className="int_values" role="button" data-button-name="AC" onClick={this.clickHandler} onKeyDown={this.clickHandler} tabIndex={0}>AC</div>
+        <div className="int_values" role="button" data-button-name="+/-" onClick={this.clickHandler} onKeyDown={this.clickHandler} tabIndex={0}>+/-</div>
+        <div className="int_values" role="button" data-button-name="%" onClick={this.clickHandler} onKeyDown={this.clickHandler} tabIndex={0}>%</div>
+        <div className="arithmetic_operand" role="button" data-button-name="÷" onClick={this.clickHandler} onKeyDown={this.clickHandler} tabIndex={0}>÷</div>
+        <div className="int_values" role="button" data-button-name="1" onClick={this.clickHandler} onKeyDown={this.clickHandler} tabIndex={0}>1</div>
+        <div className="int_values " role="button" data-button-name="2" onClick={this.clickHandler} onKeyDown={this.clickHandler} tabIndex={0}>2</div>
+        <div className="int_values " role="button" data-button-name="3" onClick={this.clickHandler} onKeyDown={this.clickHandler} tabIndex={0}>3</div>
+        <div className="arithmetic_operand" role="button" data-button-name="x" onClick={this.clickHandler} onKeyDown={this.clickHandler} tabIndex={0}>x</div>
+        <div className="int_values " role="button" data-button-name="4" onClick={this.clickHandler} onKeyDown={this.clickHandler} tabIndex={0}>4</div>
+        <div className="int_values " role="button" data-button-name="5" onClick={this.clickHandler} onKeyDown={this.clickHandler} tabIndex={0}>5</div>
+        <div className="int_values " role="button" data-button-name="6" onClick={this.clickHandler} onKeyDown={this.clickHandler} tabIndex={0}>6</div>
+        <div className="arithmetic_operand" role="button" data-button-name="-" onClick={this.clickHandler} onKeyDown={this.clickHandler} tabIndex={0}>-</div>
+        <div className="int_values " role="button" data-button-name="7" onClick={this.clickHandler} onKeyDown={this.clickHandler} tabIndex={0}>7</div>
+        <div className="int_values " role="button" data-button-name="8" onClick={this.clickHandler} onKeyDown={this.clickHandler} tabIndex={0}>8</div>
```

Comment:
**Optional Changes**
- [ ] There is too much repetition going on here. You may extract the button into its own Class Component and also consider storing the button names in an array. In that where, you can use `array.map` to dynamically add them to the DOM. In this way, we will be adhering to the `DRY` princples.